public/app/features/alerting/unified/components/receivers/TemplateForm.tsx

```diff
@@ -167,6 +173,22 @@ export const TemplateForm = ({ originalTemplate, prefill, alertmanager }: Props)
 
   const handleTemplateGenerated = (template: string) => {
     setValue('content', template);
+    setAiGeneratedTemplate(true);
```

Comment:
Similar concern as before. We start to mix OSS and enterprise code here. I'm worried that it will be very easy to break the enterprise code without even noticing. Couldn't we track that in enterprise?

Reply:
I see moving the the feedback button to enterprise, but not this kind of code where we are deciding in the consumer (OSS) how to use these feedback buttons (the actions to take, etc...)

Reply:
Ok, I think I found a way

Reply:
let's move for now, all the code from feedback button in this pr, and let's figure out in a sepatate pr how to handle the same approach for the generation state